### ISSUE TITLE
Is this repo still used?

### ISSUE THREAD

**Issue by ahmedwahdan** [libmpack/libmpack#43]
I see the implementation has not been updated for a while now. Is it stable?

**Comment by tarruda** [comment]
Yes, this should be a complete implementation of the [current spec](https://github.com/msgpack/msgpack/blob/master/spec.md)

(Feel free to report any inconsistencies with the spec)

**Comment by ahmedwahdan** [comment]
Thanks tarruda!

I used nanopb and there we just use the encode, decode API, why we don't have the same APIs?

**Comment by tarruda** [comment]
The goal of this library is to give more control to the user. A simple encode/decode would probably need to allocate memory for the user, which is not ideal when you want to expose to another language that handles its own allocation.

See the [rationale](https://github.com/libmpack/libmpack#rationale) for more info.
